### PR TITLE
Pricing table, FAQ & Compute

### PR DESCRIPTION
**Pull request by avinder-red-crackle**

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64a431546c610a1c35a86664--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64a5c167ad2f6c1f80e480d5--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64a5c4a8ad2f6c2230e4814a--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64a5e55784d273059ff1ac06--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64a5eefbcfcec0098531e69b--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64a7271598f7557b4f98cc6a--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
@avinder-red-crackle Can you resolve these conflicts today so we can push it tomorrow if we need to?

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64aef93ce51d701205c7ab1d--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64af16e1deb6762641be09a7--ephemeral-narwhal-3ac3b2.netlify.app

File: website/_data/tier-2-pricing.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- Up to 5 users
+- XS, S, and M satellites
+- 24 hour max build time
+- Community support
```
Comment (vladaionescu) [inline website/_data/tier-2-pricing.yml:4]:
```suggestion
- Community support
```

File: website/_data/tier-1-pricing.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- <a class="underline" href="/earthly-satellites">Satellites</a>
+- <a class="underline" href="/earthly-ci">CI</a>
+- Log Sharing
+- <a class="underline" href="https://docs.earthly.dev/earthly-cloud/cloud-secrets">Secrets</a>
+- XS and S satellites
+- 8 days cache retention
+- 2 hour max build time
+- x86 satellites only
```
Comment (vladaionescu) [inline website/_data/tier-1-pricing.yml:8]:
```suggestion
- x86 satellites only
```

**Review by vladaionescu — COMMENTED** [review]
Let's remove these as they came up in a user test session and they seemed distracting.

**Review by thtmnisamnstr — APPROVED** [review]

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64b51354c86df1043716d90b--ephemeral-narwhal-3ac3b2.netlify.app" >> $GITHUB_STEP_SUMMARY

**Pull request merged**